android/src/com/mapswithme/maps/editor/ViralFragment.java

```diff
@@ -48,16 +49,13 @@ public Dialog onCreateDialog(Bundle savedInstanceState)
     mViral = (TextView) root.findViewById(R.id.viral);
     initViralText();
     mViral.setText(sViralText);
-    // TODO set rank with correct text.
     root.findViewById(R.id.tell_friend).setOnClickListener(new View.OnClickListener()
     {
       @Override
       public void onClick(View v)
       {
-        dismiss();
```

Comment:
Убираем потом, когда выбираем тип шаринга.
> On Mar 25, 2016, at 7:49 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/ViralFragment.java <https://github.com/mapsme/omim/pull/2541#discussion_r57462550>:
> 
> >      root.findViewById(R.id.tell_friend).setOnClickListener(new View.OnClickListener()
> >      {
> >        @Override
> >        public void onClick(View v)
> >        {
> > -        dismiss();
> Убирать диалог точно не нужно?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2541/files/a197ad50eaf33fe5a3da6c5f99b49ec2c4290ab3#r57462550>